lib/autokey/gtkui/configwindow.py

```diff
@@ -1330,13 +1340,35 @@ def on_rename(self, widget, data=None):
         dlg.destroy()
 
     def on_treeWidget_row_activated(self, widget, path, viewColumn, data=None):
+        """This function is called when a row is double clicked"""
         if widget.row_expanded(path):
             widget.collapse_row(path)
         else:
             widget.expand_row(path, False)
 
     def on_treeWidget_row_collapsed(self, widget, tIter, path, data=None):
         widget.columns_autosize()
+        p = path.to_string()
+        path_len = len(p)
```

Comment:
This needs to be a new function with a descriptive name. I can't tell what this is doing at a glance.